pkg/kinflate/util/labelsandannotationsconfig.go

```diff
@@ -20,6 +20,8 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
+// DefaultLabelsPathConfigs is the default configuration for mutating labels and
+// selector fields for native k8s APIs.
 var DefaultLabelsPathConfigs = []PathConfig{
```

Comment:
It seems to me these can be package private.  They could even be embedded in the constructors suggested above.